app/views/govuk_publishing_components/components/_single_page_notification_button.html.erb

```diff
@@ -0,0 +1,31 @@
+<%
+  page ||= ''
+  data_attributes ||= {}
+  margin_top ||= 3
+  base_path ||= nil
+  margin_bottom ||= 3
+  already_subscribed ||= false
+  text ||= already_subscribed ? t('components.single_page_notification_button.unsubscribe_text') : t('components.single_page_notification_button.subscribe_text')
+
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new({
+    margin_top: margin_top,
+    margin_bottom: margin_bottom
+  })
+
+  wrapper_classes = %w(gem-c-single-page-notification-button govuk-!-display-none-print)
+  wrapper_classes << (shared_helper.get_margin_top)
```

Comment:
Also is it necessary to wrap the call to shared_helper in brackets? Will this work?

```
wrapper_classes << shared_helper.get_margin_top
```

Alternatively, could you combine both lines into

```
wrapper_classes << (shared_helper.get_margin_top shared_helper.get_margin_bottom)
```

Reply:
Yeah if I'm honest I'm not sure what I was thinking with the `margin-top` on this. It's been a couple of months since I first opened a draft PR for this... as things stand right now I don't think we should need a margin-top on this element so I've removed it.